docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -1,332 +1,407 @@
 
-According to the Auto-device selection logic from the previous section, tell the OpenVINO Runtime 
-to use the most suitable device from available devices as follows:
+1. Check which supported devices are available. 
+2. Check the precision of the input model (for detailed information on precisions read more on the [OPTIMIZATION_CAPABILITIES metric](../IE_PLUGIN_DG/Plugin.md)) 
+3. From the priority list, select the first device capable of supporting the given precision. 
+4. If the network’s precision is FP32 but there is no device capable of supporting it, offload the network to a device supporting FP16. 
 
-@snippet snippets/AUTO2.cpp part2
+@sphinxdirective
++----------+-------------------------------------------------+-------------------------------------+
+| Choice   | | Supported                                     | | Supported                         |
+| Priority | | Device                                        | | model precision                   |
++==========+=================================================+=====================================+
+| 1        | | dGPU                                          | FP32, FP16, INT8, BIN               |
+|          | | (e.g. Intel® Iris® Xe MAX)                    |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 2        | | VPUX                                          | INT8                                |
+|          | | (e.g. Intel® Movidius® VPU 3700VE)            |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 3        | | iGPU                                          | FP32, FP16, BIN,                    |
+|          | | (e.g. Intel® UHD Graphics 620 (iGPU))         |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 4        | | Myriad                                        | FP16                                |
+|          | | (Intel® Neural Compute Stick 2 (Intel® NCS2)) |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 4        | | IA CPU                                        | FP32, FP16, INT8, BIN               |
+|          | | (e.g. Intel® Core™ i7-1165G7)                 |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+@endsphinxdirective
 
-You can also use the Auto-device plugin to choose a device from a limited choice of devices, in this example CPU and GPU:
+To put it simply, when loading the network to the first device on the list fails, AUTO will try to load it to the next device in line, until one of them succeeds. For example: 
```

Comment:
Please be consistent: AUTO, Auto-Device, Auto-device? The label "AUTO" is introduced just in the **Using the Auto-Device Plugin** section.